doc/source/changes/version_0_25.rst.inc

```diff
@@ -102,6 +102,30 @@
     arr1 -> b, c, a
     arr2 -> b, a
 
+* implemented unary operations on Session, which means one can negate all arrays in a Session or take the absolute
+  value of all arrays in a Session without writing an explicit loop for that.
+
```

Comment:
Sessions are not widely used for now (except to do Session(...).to_globals() + I/O), so I guess the example with abs is enough.